core-blocks/test/helpers/index.js

```diff
@@ -5,14 +5,16 @@ import { render } from 'enzyme';
 import { noop } from 'lodash';
 
 /**
- * Internal dependencies
+ * WordPress dependencies
  */
 import {
 	createBlock,
 	getBlockType,
 	registerBlockType,
-} from '../..';
-import { BlockEdit } from '../../block-edit';
+} from '@wordpress/blocks';
+
+// Hack to avoid the wrapping HoCs.
+import { BlockEdit } from '../../../blocks/block-edit';
```

Comment:
I still have a failing test, I suspected it's happening because of this but couldn't figure out exactly. Any idea @gziolo

Reply:
`ContrastChecker` wasn't exported. Fixed with https://github.com/WordPress/gutenberg/pull/6351/commits/69185957c1b1b43c1e59f883277337cc52c3e834  :)

Reply:
Thanks ❤️